pkg/cloudprovider/utils.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudprovider
+
+import (
+	"fmt"
+	"net"
+	"strings"
+)
+
+const (
+	defaultLBSourceRange = "0.0.0.0/0"
+)
+
+// TODO: Should I move this into utils/sets ?  Even though it isn't generated?
```

Comment:
OK ... will remove comment.  We can always move later if we add sets support for non-map-key types